src/components/redux/books/booksSlice.js

```diff
@@ -0,0 +1,49 @@
+// import { createSlice } from '@reduxjs/toolkit';
+// import bookList from '../../home/bookLists';
+
+// const initialState = {
+//   bookList,
+// };
+
+// const bookSlice = createSlice({
+//   name: 'book',
+//   initialState,
+//   reducers: {
+//     addBook: (state, action) => {
+//       const newBook = action.payload;
+//       state.bookList.push(newBook);
+//     },
+//     removeBook: (state, action) => {
+//       const itemId = action.payload;
+//       state.bookList = state.bookList.filter((item) => item.id !== itemId);
+//     },
+//   },
+// });
+
+// export const { removeBook, addBook } = bookSlice.actions;
+// export default bookSlice.reducer;
+
+import { createSlice } from '@reduxjs/toolkit';
+import bookList from '../../home/bookLists';
+
+const initialState = {
+  bookList,
```

Comment:
Nice job so far, just as it is required in this milestone, the initial state should contain an empty array of books😉. Kindly do well to make this correction😉.

<img width="638" alt="image" src="https://github.com/riskmaris/bookstore/assets/64643995/3121fab2-8fed-43d4-8837-79842ed6ad14">